src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -3484,6 +3484,7 @@ public void testNetworks() throws Exception {
     final IpamConfig ipamConfig =
         IpamConfig.create("192.168.0.0/24", "192.168.0.0/24", "192.168.0.1");
     final Ipam ipam = Ipam.builder()
+        .driver("default")
```

Comment:
Not certain but https://travis-ci.org/spotify/docker-client/jobs/211368829 failing might be related to this?

> Tests in error: 
  testCreateServiceWithNetwork(com.spotify.docker.client.DefaultDockerClientTest): Request error: POST unix://localhost:80/networks/create: 500, body: {"message":"rpc error: code = 3 desc = driver name: if driver is specified name is required"}(..)